Envoy.blade.php

```diff
@@ -210,6 +213,43 @@
     fi
 @endtask
 
+@task('deploy:cronjobs')
+    IS_CRONTAB_MANAGED_BY_ENVOY=$(crontab -l | grep -c 'MANAGED BY ENVOY') || true
```

Comment:
When looking at it with Simon, we realize that there are some websites that share the same user. We could have a situation where the deployment of each website overwrites the cron configuration of the other one.

Maybe we should add a unique identifier in the message "MANAGED BY ENVOY"?